eglot-x.el

```diff
@@ -364,6 +373,13 @@ connections."
                (old (if (eq exp eglot--{}) '() exp))
                (new (plist-put old :localDocs t)))
           (setq capabilities (plist-put capabilities :experimental new))))
+      (when (and eglot-x-enable-inactive-code
+		 (cl-find "clangd" (process-command
```

Comment:
I'd rather not limit this to clangd.  Other servers might implement the inactiveRegions capability in the future.